src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -141,44 +141,57 @@ public static TypedData ToRpc(this object value, ILogger logger)
                 if (request.Body != null && request.ContentLength > 0)
                 {
                     object body = null;
-                    string rawBody = null;
+                    object rawBodyBytes = null;
```

Comment:
why do you need rawBodyBytes?

Reply:
Shouldn't be needed. I think it just got left over while I was iterating on the code logic.